styles/styles.css

```diff
@@ -68,13 +68,21 @@
   }
 }
 
+html,
+body {
+  height: 100%; /* For bottom aligned footer */
+}
+
 body {
   background-color: var(--body-background-color);
   font-family: var(--body-font-family);
   font-size: var(--body-font-size-m);
   color: var(--body-color);
   margin: 0;
   padding: 0;
+  /* For bottom aligned footer */
+  display: flex;
```

Comment:
Might be ok but this definitively have some impact on the whole styling. @davidnuescheler WDYT ?

Reply:
yeah, seems a bit scary... but i don't think it is an issue.

Reply:
...is there any reason why the default `block` wouldn't do the trick though?

Reply:
@davidnuescheler  Here is a pretty good explanation https://css-tricks.com/couple-takes-sticky-footer/#there-is-flexbox

Adding `display:flex` to the body makes it so the containers inside of it `header`, `content`, `footer` can be controlled with more detail. In this case, forcing the footer to always sit at the very bottom of the page, even with a variable height footer.

Maybe this isn't even an issue. I just noticed it on the index page, that has minimal content, the footer was floating up from the bottom. This was one way to solve for that.